Dockerfile

```diff
@@ -0,0 +1,48 @@
+# RethinkRobotics/baxter-simulator:kinetic
```

Comment:
A running example is here: https://hub.docker.com/r/davetcoleman/baxter_simulator/builds/